src/main/java/org/broadinstitute/hellbender/engine/FeatureManager.java

```diff
@@ -66,12 +61,16 @@
     private static final Class<FeatureInput> FEATURE_ARGUMENT_CLASS = FeatureInput.class;
 
     /**
-     * At startup, walk through the packages in CODEC_PACKAGES, and save any (concrete) FeatureCodecs discovered
+     * At startup, walk through the packages in codec packages, and save any (concrete) FeatureCodecs discovered
      * in DISCOVERED_CODECS
      */
     static {
+
+        // Get our configuration:
+        final GATKConfig config = ConfigCache.getOrCreate( GATKConfig.class );
+
         final ClassFinder finder = new ClassFinder();
-        for ( final String codecPackage : CODEC_PACKAGES ) {
+        for ( final String codecPackage : config.codec_packages() ) {
```

Comment:
This may become more complicated if we add a way to enable/disable sub-packages/classes, but I guess that it correspond to a different issue (should I open an issue to point out this requirement?)